src/cli_plugin/install/kibana.js

```diff
@@ -46,14 +46,14 @@ export async function rebuildCache(settings, logger) {
 }
 
 export function assertVersion(settings) {
-  if (!settings.plugins[0].version) {
+  if (!settings.plugins[0].kibanaVersion) {
```

Comment:
This error can also be solved by adding a `kibana.version` property, right? We should probably update the Error to state something along these lines: "Plugin package.json is missing both a version property (required) and a kibana.version property (optional)."